packages/slate-editor/src/extensions/inline-contacts/InlineContactsExtension.tsx

```diff
@@ -0,0 +1,48 @@
+import type { Extension } from '@prezly/slate-commons';
+import { createDeserializeElement } from '@prezly/slate-commons';
+import { CONTACT_NODE_TYPE, isContactNode } from '@prezly/slate-types';
+import React from 'react';
+
+import { composeElementDeserializer } from '#modules/html-deserialization';
+
+import {
+    normalizeContactInfoAttributes,
+    normalizeContactNodeAttributes,
+    parseSerializedElement,
+} from '../press-contacts/lib';
```

Comment:
Throughout this PR I am reusing a bunch of code from the `press-contacts` extension. I wasn't sure if it's worth it and where exactly to put this common functionality.

Assuming that we might possibly merge these two extensions into one in the future, is it okay like this (for now)?

Reply:
Agree with reusing them. :+1: 

I'd only suggest avoiding deep imports and explicitly exporting these via the `press-contacts` modue index.ts file.